gravitee-apim-console-webui/src/management/api/portal/plans/api-portal-plan-list/api-portal-plan-list.component.spec.ts

```diff
@@ -286,6 +295,13 @@ describe('ApiPortalPlanListComponent', () => {
 
       await loader.getHarness(MatButtonHarness.with({ selector: '[aria-label="View the plan details"]' })).then((btn) => btn.click());
 
+      await loader.getHarness(MatButtonHarness.with({ selector: '[aria-label="Add new plan"]' })).catch((reason) => {
```

Comment:
WDYT to juste have a .length and expect 0 ? with getAllHarness and same `with` ?

Reply:
done